### PR TITLE
[13.0][MIG] partner_deduplicate_filter

### PR DESCRIPTION
**Pull request by joao-p-marques**
Superseeds https://github.com/OCA/partner-contact/pull/819

Just rebased and improved user documentation a little.

@Tecnativa
TT19783

ping @pedrobaeza

### CODE REVIEW

**Comment by victoralmau** [comment]
Please @joao-p-marques check travis.

**Comment by kevinkhao** [comment]
coincidentially was just about to do the same thing

LGTM except precommit

thanks

**Review by kevinkhao — APPROVED** [review]

**Comment by joao-p-marques** [comment]
Thank you @victoralmau @kevinkhao 
Just updated with pre-commit, I think it should fix CI.

File: partner_deduplicate_filter/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* Pedro M. Baeza <pedro.baeza@tecnativa.com>
+* Vicent Cubells <vicent.cubells@tecnativa.com>
+* Luis M. Ontalba <luis.martinez@tecnativa.com>
+* Victor M.M. Torres <victor.martin@tecnativa.com>
```
Comment (victoralmau) [inline partner_deduplicate_filter/readme/CONTRIBUTORS.rst]:
```suggestion
* `Tecnativa <https://www.tecnativa.com>`_:

  * Pedro M. Baeza
  * Vicent Cubells
  * Luis M. Ontalba
  * Victor M.M. Torres
```

File: partner_deduplicate_filter/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+# Copyright 2019 Victor M.M Torres <victor.martin@tecnativa.com>
```
Comment (victoralmau) [inline partner_deduplicate_filter/__manifest__.py]:
```suggestion
# Copyright 2016 Tecnativa - Pedro M. Baeza
# Copyright 2017 Tecnativa - Vicent Cubells
# Copyright 2019 Tecnativa - Victor M.M Torres
```

File: partner_deduplicate_filter/tests/test_partner_deduplicate_filter.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
```
Comment (victoralmau) [inline partner_deduplicate_filter/tests/test_partner_deduplicate_filter.py]:
```suggestion
# Copyright 2016 Tecnativa - Pedro M. Baeza
```

File: partner_deduplicate_filter/wizards/partner_merge.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
```
Comment (victoralmau) [inline partner_deduplicate_filter/wizards/partner_merge.py]:
```suggestion
# Copyright 2016 Tecnativa - Pedro M. Baeza
```

File: partner_deduplicate_filter/wizards/partner_merge_view.xml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+     Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+     Copyright 2019 Victor M.M Torres <victor.martin@tecnativa.com>
```
Comment (victoralmau) [inline partner_deduplicate_filter/wizards/partner_merge_view.xml]:
```suggestion
<!-- Copyright 2016 Tecnativa - Pedro M. Baeza
     Copyright 2017 Tecnativa - Vicent Cubells
     Copyright 2017 Tecnativa - Victor M.M Torres
```

File: partner_deduplicate_filter/i18n/es.po [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+msgid "Parent company not set"
+msgstr "La compañía padre no establecida"
+
+#. module: partner_deduplicate_filter
+#: model:ir.model.fields,field_description:partner_deduplicate_filter.field_base_partner_merge_automatic_wizard__exclude_parent
+msgid "Parent company set (Contacts)"
+msgstr "La compañía padre establecida (Contactos)"
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_filter/i18n/es.po]:
```suggestion
```

File: partner_deduplicate_filter/i18n/de.po [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+msgid "Parent company not set"
+msgstr "Übergeordnetes Unternehmen nicht gepflegt"
+
+#. module: partner_deduplicate_filter
+#: model:ir.model.fields,field_description:partner_deduplicate_filter.field_base_partner_merge_automatic_wizard__exclude_parent
+msgid "Parent company set (Contacts)"
+msgstr "Übergeordnetes Unternehmen gesetzt (Kontakte)"
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_filter/i18n/de.po]:
```suggestion
```

File: partner_deduplicate_filter/i18n/hr.po [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+msgid "Parent company not set"
+msgstr ""
+
+#. module: partner_deduplicate_filter
+#: model:ir.model.fields,field_description:partner_deduplicate_filter.field_base_partner_merge_automatic_wizard__exclude_parent
+msgid "Parent company set (Contacts)"
+msgstr ""
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_filter/i18n/hr.po]:
```suggestion
```

File: partner_deduplicate_filter/i18n/nl_NL.po [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+msgid "Parent company not set"
+msgstr "Moederbedrijf niet ingesteld"
+
+#. module: partner_deduplicate_filter
+#: model:ir.model.fields,field_description:partner_deduplicate_filter.field_base_partner_merge_automatic_wizard__exclude_parent
+msgid "Parent company set (Contacts)"
+msgstr ""
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_filter/i18n/nl_NL.po]:
```suggestion
```

File: partner_deduplicate_filter/i18n/sk.po [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+msgid "Parent company not set"
+msgstr ""
+
+#. module: partner_deduplicate_filter
+#: model:ir.model.fields,field_description:partner_deduplicate_filter.field_base_partner_merge_automatic_wizard__exclude_parent
+msgid "Parent company set (Contacts)"
+msgstr ""
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_filter/i18n/sk.po]:
```suggestion
```

File: partner_deduplicate_filter/i18n/tr.po [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+msgid "Parent company not set"
+msgstr "Üst firma Ayarlanmamışsa"
+
+#. module: partner_deduplicate_filter
+#: model:ir.model.fields,field_description:partner_deduplicate_filter.field_base_partner_merge_automatic_wizard__exclude_parent
+msgid "Parent company set (Contacts)"
+msgstr ""
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_filter/i18n/tr.po]:
```suggestion
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Review by victoralmau — APPROVED** [review]

**Pull request merged**